File: BEE 1051.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double num = input.nextDouble();
        if (num >= 0 && num <= 2000){
            System.out.printf("Isento\n");
        }
        else if (num > 2000 && num <= 3000){
            System.out.printf("R$ %.2f\n", ((num - 2000) * (8.0 / 100)));
        }
        else if (num > 3000 && num <= 4500){
            double ans = (1000 * (8.0 / 100)) + ((num - 3000) * (18.0 / 100));
            System.out.printf("R$ %.2f\n", ans);
        }
        else{
            double ans = (1000 * (8.0 / 100)) + (1500 * (18.0 / 100)) + ((num - 4500) * (28.0 / 100));
            System.out.printf("R$ %.2f\n", ans);
        }
    }
}

File: BEE 1051.cpp
#include <bits/stdc++.h>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(2) << (x)

using namespace std;

int main(){
    double num;
    cin >> num;
    if (num >= 0 && num <= 2000){
        cout << "Isento" << endl;
    }
    else if (num > 2000 && num <= 3000){
        cout << "R$ " << FIXED_FLOAT((num - 2000) * (8.0 / 100)) << endl;
    }
    else if (num > 3000 && num <= 4500){
        double ans = (1000 * (8.0 / 100)) + ((num - 3000) * (18.0 / 100));
        cout << "R$ " << FIXED_FLOAT(ans) << endl;
    }
    else{
        double ans = (1000 * (8.0 / 100)) + (1500 * (18.0 / 100)) + ((num - 4500) * (28.0 / 100));
        cout << "R$ " << FIXED_FLOAT(ans) << endl;
    }
}